halloffame.css

```diff
@@ -0,0 +1,67 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    background-color: #0bb016;
+    /*background-image: url("");*/
+    background-size: cover;
+    width: 100%;
+    background-repeat: no-repeat;
+    font-family: 'Exo 2', sans-serif;
+
+}
+.navTop {
+    background-image: url("https://image.freepik.com/free-photo/green-grass-field-background-soccer-football-sports_64749-1823.jpg");
+    padding: 5px;
+    width: 100%;
+    margin: 0 auto;
+    color: #2a324b;
+    border-style: solid;
+    border-width: 1px;
+    border-color: #2a324b;
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+h1 {
+    margin-left: 70px;
+    font-size: 70px;
+}
+
+h3 {
+    margin-left: 600px;
+    font-size: 80px;
+    }
+
+header {
+    position: fixed;
+    top:0;
+    right:0;
+    display: inline-block;
+    list-style: none;
+    cursor:pointer;
+    padding:15px;
+}
+
+.Players {
```

Comment:
Inconsistent class name casing here - suggest make all lower case in line with other classes